week-4/Homework/mandatory/3-translator.js

```diff
@@ -28,9 +28,28 @@ It should default to English if the language is not in your object of languages,
 or in the event of an invalid input.
 */
 
+
+
 function greet(language) {
-  //write your code here
-  return languages[language];
+
+  if (typeof language === "string") {
+    let lang = language.toLowerCase();
+    if (Object.keys(languages).includes(lang)) {
+      return languages[lang];
+    } else {
+      return languages.english;
+    }
+  } else {
+    return "that's not a language";
+  }
+}  
+  
+
+console.log(greet("irish"));
+console.log(greet("FINNish"));
+console.log(greet("russian"));
+console.log(greet(6));
+return languages[language];
 }
```

Comment:
This return outside the function can be removed as it doesn't do anything.

Reply:
that's not what I have in my file. there are no lines 32,33 and 52,53. Why are they shown here?